### ISSUE TITLE
Telephone with errors example has incorrect autocomplete attribute

### ISSUE THREAD

**Issue by emilymaylawes** [ONSdigital/design-system#2735]
The example [Example phone number error](https://service-manual.ons.gov.uk/design-system/patterns/phone-numbers/example-phone-number-error) has the HTML attribute `autocomplete="telephone"`. This has been done due to the WCAG criteria [1.3.5 Identify input purpose](https://www.w3.org/WAI/WCAG21/Understanding/identify-input-purpose.html) where the only way of programmatically identifying form fields for user input is via the `autocomplete` attribute, however the HTML spec is [strict in which string values it allows](https://www.w3.org/TR/WCAG21/#input-purposes). For telephone numbers, the correct `autocomplete` attribute is `tel`.

This can successfully be seen at this [first example with no errors](https://service-manual.ons.gov.uk/design-system/components/input/example-input-telephone).

This is currently a failure for the Accessibility Statement.
